e2e-tests/cypress/integration/PageObjects/PO_EligibilityDecisionDate.js

```diff
@@ -0,0 +1,115 @@
+class PO_EligibilityDecisionDate {
+    
+    navigatetoEligDatePageURL() {
+        cy.visit("https://appeals-dev.planninginspectorate.gov.uk/eligibility/decision-date")
+    }
+
+    validatePageTitle() {
+        //cy.title().should('eq', 'Local Planning authority details') -> Actual, delete the below line
+        cy.title().should('eq', 'GOV.UK - The best place to find government services and information')
+    }
+
+    validateHeaderLogo() {
+        const headerLogo = cy.get('.govuk-header__logotype')
+        assert.exists(headerLogo, 'GOV UK Logo exists')
+    }
+
+    pageHeaderlink() {
+        //const headerLink = cy.get('a[href="Appeal a householder planning decision"]')
+        const headerLink = cy.get('.govuk-header__content > .govuk-header__link')
+        assert.exists(headerLink, 'Appeal a householder planning decision exists')
+    }
+
+    bannerText() {
+        cy.get('.govuk-tag govuk-phase-banner__content__tag'), should('eq', "beta")
```

Comment:
should be `.should(` - seems like this will cause the tests to fail

Reply:
i've taken out the WIP features so i can just get one test in that passes; a lot of these spec files are things i've not even opened yet..

Reply:
(also seen Simon's observation on volume of commented out code; just going to go and work out which of these specs are in use and clean up a little + noticed a couple of his initial comments i've not resolved yet.. -> quickly look at those)

Reply:
have had a bit of a tidy up-  what's left isn't necessarily 'great' but i've cleared out the things i'm not currently running.
will be bringing the tests back in as another branch as Karuna and I flesh them out